pkg/controller/pxcbackup/controller.go

```diff
@@ -423,18 +418,21 @@ func (r *ReconcilePerconaXtraDBClusterBackup) runBackupFinalizers(ctx context.Co
 		var err error
 		switch f {
 		case naming.FinalizerDeleteBackup:
-			if (cr.Status.S3 == nil && cr.Status.Azure == nil) || cr.Status.Destination == "" {
+			storageType := cr.Status.GetStorageType(nil)
+			if (cr.Status.S3 == nil && cr.Status.Azure == nil && storageType != api.BackupStorageFilesystem) || cr.Status.Destination == "" {
```

Comment:
didn't we agree on adding `cr.Status.Filesystem`?

Reply:
Yes, we didn't. I was confused what variables should we put in this structure. Let's decide here and I will fix.

Reply:
ok. what confused you? what needs to be decided?

Reply:
So for example for azure and aws in this structure we have 
```
type BackupStorageAzureSpec struct {
	CredentialsSecret string `json:"credentialsSecret"`
	ContainerPath     string `json:"container"`
	Endpoint          string `json:"endpointUrl"`
	StorageClass      string `json:"storageClass"`
	BlockSize         int64  `json:"blockSize"`
	Concurrency       int    `json:"concurrency"`
}

type BackupStorageS3Spec struct {
	Bucket            string `json:"bucket"`
	CredentialsSecret string `json:"credentialsSecret"`
	Region            string `json:"region,omitempty"`
	EndpointURL       string `json:"endpointUrl,omitempty"`
}
```
So in this case, what would be reasonable to add to the same structure for the fs-pvc?